mappings/net/minecraft/nbt/NbtEnd.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_2491 net/minecraft/nbt/NbtEnd
+	COMMENT Represents the NBT end value.
+	COMMENT Defines the end of an {@link NbtCompound} object during serialization,
+	COMMENT and is the type of an empty {@link NbtList}.
+	FIELD field_21032 TYPE Lnet/minecraft/class_4614;
+	FIELD field_21033 INSTANCE Lnet/minecraft/class_2491;
```

Comment:
```suggestion
	FIELD field_21033 INSTANCE Lnet/minecraft/class_2491;
		COMMENT A dummy instance of the NBT end. It will never appear in any parsed NBT
		COMMENT structure and should never be used as NBT compound or list elements.
```